### DISCUSSION TITLE
Ability to disable features.

### DISCUSSION THREAD

**Discussion by Juha83** [woocommerce/woocommerce#45075] [WooCommerce, Blocks, and Site Editing]
## What do you like about the Cart & Checkout blocks?

Woocommerce Blocks adds very nice layouts for my Shopping cart and checkout page, and it is easy to use.

## What do you think is missing from the Cart & Checkout blocks?

There should be a change to turn of the features like the stock and hide the Save block.

**Comment by nerrad** [comment]
HI there @Juha83  👋🏻

Thanks so much for taking the time to leave some feedback - much appreciated! We make note of all feedback we receive and we also close issues with the feedback as active issues are used for bugs or feature/enhancements. It does not mean that we don't note or review the feedback. Closed issues can also still be commented on.

To help us out when we review requests like yours, would you be able to clarify a bit more around what you mean by "turn off the features like the stock and hide the Save block". Sharing screenshots indicating what you are referencing helps as well! Even though the issues is closed from time to time we still review issues like yours tagged as `category:feedback` to help inform potential improvements. So providing additional detail helps with that.

**Comment by Juha83** [comment]
Hello,
I added the image to show the features which should be able to disable or enable.

Show discount ( Exp. SAVE €0.00) should be optional because it's not a feature that everyone is using however it is a great feature to have.

What I meant about the "stock", I meant Quantity selector in the shopping cart.

I know that I can disable it by using custom CSS or PHP function but I am trying not to add any extra codes.

![Option to disable](https://user-images.githubusercontent.com/62006154/121471260-05083c00-c9c8-11eb-98b4-f68579b17072.jpg)

**Comment by nerrad** [comment]
Thanks for the additional details! Would you be able to share more information around the use-cases for having no quantity selector? My question is intended to understand more about what problems removing the quantity selector solves.

**Comment by Juha83** [comment]
No worries. For example, if I'm using a bundle plugin such as WPC Product Smart Bundle. And I have the main product, one with the price and the other one is a free product. I don't want it to show that the product is counted as discounted or that customer is able to remove or change the quantity of the product.

In the WPC product Smart Bundle there is an option to turn off these features but it does not override the Woocommerce Blocks functions. I already contacted them but they said that cannot do anything about it.

Not sure if this is a common thing for the product bundles but in this case, this does not work.

**Comment by nerrad** [comment]
Thanks for sharing more details @Juha83. It's unlikely we will support this use case as a user controlled option in the blocks because it creates additional complexity around different use-cases that is best handled at extension level. It's good you've let the extension developer know about this - it will be up to them on whether they support the cart and checkout blocks or not.

**Comment by Juha83** [comment]
No problem. This is a very different way to sell products than in usual cases. Anyway, we let the bundle plugin go and using attributes and variation products from now on. So now the blocks are working just fine. Thank you for the support though.
